Tools/Create-Azure-Sentinel-Solution/createSolution.ps1

```diff
@@ -494,11 +501,20 @@ foreach ($inputFile in $(Get-ChildItem $path)) {
                                     $foundConnection = getConnectionVariableName $connectionVar
                                     if ($foundConnection) {
                                         $playbookResource.properties.api.id = "[variables('_$foundConnection')]"
+                                        if(($playbookResource.properties.parameterValues) -and ($null -ne $baseMainTemplate.variables.'playbook-ApiKey'))
```

Comment:
Since this is done at the end of both cases you should be able to just take this logic out of the conditionals and put it below.

Reply:
Yes I have taken care.